### ISSUE TITLE
请问一下数据集的事情，下载后压缩文件损坏（compressed file is damaged）

### ISSUE THREAD

**Issue by yudmoe** [kailaisun/Indoor-Depth-Completion#2]
您好，我下载了您提供的数据集后两次都有文件损坏的情况。打开之后里头只有_full后缀的文件夹，并且无法正常解压。
1.可否提供其他的下载途径，比如百度网盘之类的。您提供的下载路径经常断。
2.或者有只提供一个数据量较小的mini数据集，用于学习和调试代码。

Hello, I downloaded the dataset you provided twice and the files were damaged. After opening it, there is only a folder with the suffix _full, and it cannot be decompressed normally.
1. Can you provide other downloading methods, such as Baidu Netdisk. The download path you provided is often broken.
2. Or only provide a mini dataset with a small amount of data for learning and debugging code.

**Comment by kailaisun** [comment]
try vpn？

---- 回复的原邮件 ----
| 发件人 | ZhengYu ***@***.***> |
| 发送日期 | 2024年10月17日 17:08 |
| 收件人 | kailaisun/Indoor-Depth-Completion ***@***.***> |
| 抄送人 | Subscribed ***@***.***> |
| 主题 | [kailaisun/Indoor-Depth-Completion] 请问一下数据集的事情，下载后压缩文件损坏（compressed file is damaged） (Issue #2) |

您好，我下载了您提供的数据集后两次都有文件损坏的情况。打开之后里头只有_full后缀的文件夹，并且无法正常解压。
1.可否提供其他的下载途径，比如百度网盘之类的。您提供的下载路径经常断。
2.或者有只提供一个数据量较小的mini数据集，用于学习和调试代码。

Hello, I downloaded the dataset you provided twice and the files were damaged. After opening it, there is only a folder with the suffix _full, and it cannot be decompressed normally.

Can you provide other downloading methods, such as Baidu Netdisk. The download path you provided is often broken.
Or only provide a mini dataset with a small amount of data for learning and debugging code.

**Comment by yudmoe** [comment]
> try vpn？
> […](#)

Yes, I used a VPN, but the connection is unstable. It took me many attempts to download the dataset, and I was very disappointed when I found that the compressed file was corrupted.

**Comment by kailaisun** [comment]
you can try winrar software to repair the file

---- 回复的原邮件 ----
| 发件人 | ZhengYu ***@***.***> |
| 发送日期 | 2024年10月17日 19:00 |
| 收件人 | kailaisun/Indoor-Depth-Completion ***@***.***> |
| 抄送人 | kailai ***@***.***>,
Comment ***@***.***> |
| 主题 | Re: [kailaisun/Indoor-Depth-Completion] 请问一下数据集的事情，下载后压缩文件损坏（compressed file is damaged） (Issue #2) |

try vpn？
…

Yes, I used a VPN, but the connection is unstable. It took me many attempts to download the dataset, and I was very disappointed when I found that the compressed file was corrupted.

**Comment by yudmoe** [comment]
Oh thank you, repair really worked. I can unzip it now.

**Comment by yudmoe** [comment]
> you can try winrar software to repair the file

Thank you for updating the dataset download link.

However, the current dataset only contains the train_full and test_full folders, while the train and test folders are missing, making the pretraining code unusable. Do you have any plans to add the missing folders in the future?

Additionally, does the train_full folder include the content that was previously in the train folder? If I modify the dataset path in main_pretrain.py, will it run correctly?

**Comment by basil-hayden** [comment]
Yes, both can use train_full, but train_full has an additional raw_depth channel. So, you need to adjust the channels, referring to the changes made in lines 296 and 297 of model_mae.py.
